### ISSUE TITLE
Support for reading and writing sdout and stdin?

### ISSUE THREAD

**Issue by SijmenHuizenga** [fussybeaver/bollard#96]
Does this library support the [Client.ContainerAttach](https://godoc.org/github.com/docker/docker/client#Client.ContainerAttach) methods to read stdout and write to stdin?

**Comment by fussybeaver** [comment]
No, none of the multiplexed tty methods that are currently implemented.
